README.md

```diff
@@ -31,6 +31,21 @@ In your project directory, run `kinvey config` to set-up your project. The CLI w
 
 You can create a one-time session using the `-e` and `-p` (and optionally `--host`) flags. Set these parameters explicitly in your calls if you are unable to init with `kinvey config [host]`.
 
+### Credentials
+
+Credentials can be specified either as command line options or as environment variables.
+* command line options
```

Comment:
I think sub-sections would be better here (instead of a bullet list) so that you can move the example above 'You can create a one-time session...' down here.

An example: 

#### Command line options
Run `kinvey config` and the CLI...
You can create a one-time-session using the `-e` and `-p` (and optionally `--host`)

Example:
```kinvey config -e kinveyAccount@kinvey.com -p yourKinveyPassword`

#### Environment variables
* `KINVEY_CLI_USER` - e-mail address ...
* `KINVEY_CLI_PASSWORD` password of your...

Example:
```
export KINVEY_CLI_USER=kinveyAccount@example.com && export KINVEY_CLI_PASSWORD=yourKinveyPassword
```